packages/yourloops/components/statistics/patient-statistics.tsx

```diff
@@ -48,15 +57,25 @@ export const PatientStatistics: FunctionComponent<PropsWithChildren<PatientStati
   const cbgStatType: CBGStatType = bgSource === DatumType.Cbg ? CBGStatType.TimeInRange : CBGStatType.ReadingsInRange
   const numberOfDays = TimeService.getNumberOfDays(dateFilter.start, dateFilter.end, dateFilter.weekDays)
   const cbgSelected = bgSource === DatumType.Cbg
-  const { sensorUsage, total } = GlycemiaStatisticsService.getSensorUsage(medicalData.cbg, numberOfDays, dateFilter)
-  const sensorUsageData = {
+  const bgUnits = MGDL_UNITS || MMOLL_UNITS
+  const theme = useTheme()
+  const curentPage = window.location.href.split('/').at(-1)
+  const isTrendsPage = curentPage === 'trends'
```

Comment:
<!-- Suggestions propose improvements to the current subject. It's important to be explicit and clear on what is being suggested and why it is an improvement. Consider using patches to further communicate your intent.  -->
**🎯 suggestion (non-blocking):** ‏
```suggestion
  const location = useLocation()
  const isTrendsPage = location.pathname.includes('trends')
```